components/Story.js

```diff
@@ -34,11 +34,11 @@ export function AddToStory({ story, saveStory }) {
       </span>
 
       <div className="mt-1">
-        <textarea
-          rows={4}
+        <input
```

Comment:
Ok, basic web dev question, but when should I use a `textarea` and when should I use a text `input`?

Reply:
totally depends on the UX you're going for - an `input` will only ever be 1 line (afaik) where a `textarea` can be multi-line and expand to fit more text.

I went with an `input` here because I like the "single underline" style - it looks like a field on a paper form and that felt good here - but there's a strong argument to be made for using a textarea - I just don't know exactly how it should look and didn't have any brilliant ideas. 

super easy to switch back to a textarea if you want to encourage people to compose longer sentences - I figured we're ok with relatively shorter sentences here so the input would be ok, but very open to being wrong about that